flipperdevices/flipperzero-firmware
# Dictionary stuff: iClass keys

# What's new

- Add comments to MFC dict
- Add system picopass app

# Verification 

- Run MFC read and see no problem due to comments
- open MFC dictionary and see comments

- Enable debug and logs
- Run picopass
- Run Read Card with non-standard key
- See dictionary of keys tried

# Checklist (For Reviewer)
